code/modules/admin/topic.dm

```diff
@@ -1004,7 +1004,7 @@
 				if(!mins)
 					return
 				if(mins >= 525600) mins = 525599
-				var/reason = input(usr,"Reason?","reason","Griefer") as text|null
+				var/reason = input(usr,"Please state the reason","Reason") as message|null

```

Comment:
Eh. It's a minor thing. But I do know that myself and other admins generally found the default reason being "Griefer" to be amusing, and more than a few bans are applied with that as the reason.